website/source/guides/operations/deployment-guide.html.md

```diff
@@ -0,0 +1,292 @@
+
+Vault requires a Consul token with specific policy to limit the requests Vault can make to Consul endpoints.
+
+On a host running a Consul agent, and using a Consul management token, create a Consul client token with specific policy for Vault:
+
+```text
+CONSUL_TOKEN="6609e426-1aeb-4b0d-c302-3a7568fbc1f9"
+curl \
+    --request PUT \
+    --header "X-Consul-Token: ${CONSUL_TOKEN}" \
+    --data \
+'{
+  "Name": "Vault Token",
+  "Type": "client",
+  "Rules": "node \"\" { policy = \"write\" } service \"vault\" { policy = \"write\" } agent \"\" { policy = \"write\" }  key \"vault\" { policy = \"write\" } session \"\" { policy = \"write\" } "
+}' http://127.0.0.1:8500/v1/acl/create
+```
+
+The response includes the UUID you will use as the `token` parameter value in Vault's storage stanza configuration. An example response:
+
+```json
+{"ID":"fe3b8d40-0ee0-8783-6cc2-ab1aa9bb16c1"}
+```
+
+## Configure Vault
+
+Vault uses [documented sane defaults](/docs/configuration) so only non-default values must be set in the configuration file.
+
+Create a configuration file at /etc/vault.d/vault.hcl:
+
+```text
+sudo mkdir --parents /etc/vault.d
+sudo touch /etc/vault.d/vault.hcl
+sudo chown --recursive vault:vault /etc/vault.d
+sudo chmod 640 /etc/vault.d/vault.hcl
+```
+
+### Listener stanza
+
+The `listener` stanza configures the addresses and ports on which Vault will respond to requests.
+
+Add the below configuration to the Vault configuration file:
+
+```hcl
+listener "tcp" {
+  address       = "0.0.0.0:8200"
+  tls_cert_file = "/path/to/fullchain.pem"
+  tls_key_file  = "/path/to/privkey.pem"
+}
+```
+
+The following parameters are set for the `tcp` listener stanza:
+
+- [`address`](/docs/configuration/listener/tcp.html#address) `(string: "127.0.0.1:8200")` – Changing from the loopback address to allow external access to the Vault UI
+- [`tls_cert_file`](/docs/configuration/listener/tcp.html#tls_cert_file) `(string: <required-if-enabled>, reloads-on-SIGHUP)` - Must be set when using TLS
+- [`tls_key_file`](/docs/configuration/listener/tcp.html#tls_key_file) `(string: <required-if-enabled>, reloads-on-SIGHUP)` - Must be set when using TLS
+
+[More information about tcp listener configuration](/docs/configuration/listener/tcp.html).
+
+~> Vault should always be configured to use TLS to provide secure communication between clients and the Vault cluster. This requires a certificate file and key file be installed on each Linux host running Vault. The certificate file and key file must have permissions allowing the vault user/group to read them.
+
+### Seal stanza
+
+This is an __ENTERPRISE__ feature.
```

Comment:
Perhaps follow a style standard for calling out enterprise features, to ensure it's clear in documentation

```
~> **Enterprise Only:** Alternative sealing methods are a feature of _Vault Enterprise_.
```
(see https://raw.githubusercontent.com/hashicorp/vault/master/website/source/guides/operations/replication.html.md)

Reply:
Good point. In writing these for every product I have noticed a fair amount of inconsistency and spoke with Geoffrey about it. It sounds like Nick will be (or already is) working on a style guide for docs that specifies how we do "notes", "tips", call-outs in general and things like oss/ent feature identification in the docs. That will be welcome, but for now, I'll update this in another PR to avoid stalling this one any further.